Source/Scene/Points3DTileContent.js

```diff
@@ -148,19 +155,42 @@ define([
         }
     });
 
+    function createFeatures(content) {
```

Comment:
OK, I still think this has the potential to introduce huge overhead for naive tiles or tiles where points truly have a ton of unique values - probably true of many datasets that do not go through a cleaning pipeline.  How hard is it to add a flag as I suggested even if it defaults to `true`?